### ISSUE TITLE
Kiblitzer nodes distribution

### ISSUE THREAD

**Issue by PinYin12** [mooskagh/lczero-live#4]
Hi, i am following the world championship with renewed interest thanks to your amazing work!!. 
I was thinking that Leela default node search distribution is very good for matches between computers, but may be a little akward for kiblitzing/commenting.
Maybe devolving a little bit more nodes to secondary moves to give more interesting sidelines? 
Usually 99%+ nodes are devolved to the main line, which is a little bit "overkill" precision for human matches.
Thanks for your attention, keep up the good work!

**Comment by mooskagh** [comment]
Yeah we also noticed and wanted to adjust it, but couldn't come up immediately with good settings (and then we forgot).

There are only few games left, so likely we'll keep it as it is.

For the next event, whatever it will be, we'll surely do something about it. We also have some other changes/improvements in mind.
